framework/resources/Functional/decimal_negative/decimal_decimal20.e_tsv

```diff
@@ -0,0 +1 @@
+.*VALIDATION ERROR: Value -1234567890000000000000000000000000000000000000 overflows specified precision 38 with scale 0\..*
```

Comment:
Validation error in result files ?

Reply:
It is negative tests. We should get an error like this to prevent negative decimal scale.